dbml-homepage/docs/docs/README.md

```diff
@@ -419,6 +420,33 @@ If your enum values contain spaces or other special characters you can use doubl
         countries
     } 
 
+## Sticky Notes
+You can include sticky notes in your diagram and easily export them as png, svg, or pdf. These notes are simple to figure out without hovering over the diagram canvas.
+
+Sticky Notes can only be defined within the same scope as the tables.
```

Comment:
I know i'm not resposible for reviewing this but this line is quite confusing anh :pepe-cuoi-guong.

Reply:
lol, ye, it's quite confusing.

You can simply write like this:
`You can add sticky notes to the diagram canvas to serve as a quick reminder or to elaborate on a complex idea.`

Reply:
Any suggestions for better writing em. Since it is only work in the global scope. But I do not think global is a good word here.

Reply:
Ok thanks Thi

Reply:
Updated

Reply:
Okay I get it. Since the "Note Definition" already mentions that "Note allows users to give description for a particular DBML element." Perhaps we can bring the Sticky Notes outside, between the "Comments" and "Note Definition" to avoid users misunderstanding that the sticky notes will be defined under any element's scope

Reply:
updated